src/Fixer/Whitespace/NoSpacesInsideParenthesisFixer.php

```diff
@@ -21,6 +21,7 @@
 use PhpCsFixer\Tokenizer\Tokens;
 
 /**
+ * @deprecated in favor of SpacesInsideParenthesis fixer
```

Comment:
This is definitely not enough - please make it an `AbstractProxyFixer` to the new one (see `PhpdocNoAliasTagFixer` how it's done).